CHANGELOG.next.asciidoc

```diff
@@ -315,6 +315,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix `httpjson` input rate limit processing and documentation. {pull}[]
 - Update Filebeat compatibility function to remove processor description field on ES < 7.9.0 {pull}27774[27774]
 - Make filestream events ECS compliant. {issue}27776[27776]
+- Fix ThreatIntel MISP testcase input file to include final case. {pull}27881[27881]
```

Comment:
This being a test improvement I wouldn't add a changelog entry. Only for user-facing changes.

Reply:
Ack